mvar/MVar.mli

```diff
@@ -20,14 +20,17 @@ end
 
 module type SCHED = sig
   type 'a cont
+
```

Comment:
Is this and further spacing edits necessary? The additional space messes up documentation generation. The documentation comments `(** ... *)` must be attached to specific items in the `mli` file for it to work correctly.